scripts/parse_footpedal_anns.py

```diff
@@ -14,24 +13,27 @@ def main(args):
     :param args.image_path: Absolute path to the 'images' folder as is output by the exploded ros bag script
     :param args.ann_path: Absolute path to the annotation_event_data.txt file as is output by the exploded ros bag
                           script
-    :param args.rosbag_name: A string containing the name of the rosbag that is being processed
+    :param args.rosbag_name: A string containing the name of the rosbag that is being processed. This is used to 
+                             create a relative filepath to the exploded rosbag image folder for evaluation.
     :param args.activity: Only used when parsing directly to the PTG labels, this is for when a recording only contains
-                     a single activity being performed repeatedly. the parameter is a string of the name of the activity
-    :param args.ptg_out_file: The absolute path of the feather file containing the PTG formatted data will be output to
+                     a single activity being performed repeatedly. The parameter is a string of the name of the activity
+    :param args.ptg_out_file: The absolute path of the feather file will will output that will contain the PTG data format.
```

Comment:
Duplicate use of "will". Maybe the first one was supposed to be "we"?
```suggestion
    :param args.ptg_out_file: The absolute path of the feather file will output that will contain the PTG data format.
```